docs/blog/posts/2023-05-24-latest_developments.md

```diff
@@ -0,0 +1,137 @@
+---
+date: 2022-07-14
+authors:
+  - ross-k
+  - robin-l
+categories:
+  - Feature Updates
+---
+
+# Splink Updates - July 2023
+
+** :new: Welcome to the Splink Blog! :new: **
+
+Its hard to keep up to date with all of the new features being added to Splink, so we have launched this blog to share a round up of latest developments every few months.
+
+So, without further ado, here are some of the highlights from the first half of 2023!
+
+<!-- more -->
+
+Latest Splink version: [v3.9.4](https://github.com/moj-analytical-services/splink/releases/tag/v3.9.4)
+
+
+## :rocket: Massive speed gains in EM training
+
+There’s now an option to make EM training much faster - in one example we’ve seen at 1000x fold speedup.  Kudos to external contributor [@aymonwuolanne](https://github.com/moj-analytical-services/splink/pull/1369) from the Australian Bureau of Statistics! 
+
+To make use of this, set the `estimate_without_term_frequencies` parameter to True, for example:
```

Comment:
Maybe ',' -> ';' here? Initially read as though setting the parameter to True is one example of how to do this

Reply:
Changed